app/clients/file_client.rb

```diff
@@ -13,7 +13,7 @@ class FileClient
   # - the contents of the file.
 
   def self.save(user_id, folder, filename, content)
-    filepath = self.filepath user_id, folder, filename
+    filepath = filepath user_id, folder, filename
```

Comment:
Disregard that, I was doing it wrong. The broken tests are indeed reproducible here.